### PR TITLE
Mention extensions and libjpeg8-dev

### PR DESCRIPTION
**Pull request by aanriot**
A few additions, feel free to reword them.

### CODE REVIEW

**Comment by aanriot** [comment]
Hello Antoine,

Any news ? :-)

**Comment by tonioo** [comment]
Hey, sorry for the delay.
Actually, the --upgrade option is not a good idea since it will try to update all dependencies, including Django.

File: doc/getting_started/upgrade.rst [hunk 1]
```diff
@@ -67,6 +73,9 @@ To use this feature, add the following configuration into the ``settings.py`` fi
       "DIGITS": 1
   }
 
+``pip`` users will need jpeg libraries in order to upgrade extensions. To install them on Debian / Ubuntu, you can run::
+
+  $ apt-get install libjpeg8-dev
```
Comment (tonioo) [inline doc/getting_started/upgrade.rst]: Do you know the dependency which requires this library ?

**Comment by aanriot** [comment]
No, sorry, but it's listed here: https://github.com/modoboa/modoboa-installer/issues/1
Feel free to close the issue, I will reopen it if I get the information.
Thanks a lot.

**Pull request closed without merging**